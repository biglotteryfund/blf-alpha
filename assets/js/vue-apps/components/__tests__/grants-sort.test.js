/**
 * @jest-environment jsdom
 */
/* eslint-env jest */
import { shallowMount } from '@vue/test-utils';
import GrantsSort from '../grants-sort.vue';

describe('GrantsSort', () => {
    it.skip('should render', () => {
        const sort = {
            defaultSort: 'awardDate|desc',
            activeSort: 'amountAwarded|asc',
            sortOptions: [
                {
                    label: 'Most recent',
                    value: 'awardDate|desc'
                },
                {
                    label: 'Oldest first',
                    value: 'awardDate|asc'
                },
                {
                    label: 'Lowest amount first',
                    value: 'amountAwarded|asc'
                },
                {
                    label: 'Highest amount first',
                    value: 'amountAwarded|desc'
                }
            ]
        };

        const wrapper = shallowMount(GrantsSort, {
            propsData: { sort }
        });

        expect(wrapper.element).toMatchSnapshot();
    });
});
