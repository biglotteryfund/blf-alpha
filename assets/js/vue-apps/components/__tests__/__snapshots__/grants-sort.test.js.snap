// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GrantsSort should render 1`] = `
<div
  class="sort-controls"
>
  <label
    class="sort-controls__label"
    for="field-sort"
  >
    
        Ordered by
    
  </label>
   
  <select
    class="sort-controls__select"
    id="field-sort"
    name="sort"
  >
    <option
      value="awardDate|desc"
    >
      
            Most recent
        
    </option>
    <option
      value="awardDate|asc"
    >
      
            Oldest first
        
    </option>
    <option
      value="amountAwarded|asc"
    >
      
            Lowest amount first
        
    </option>
    <option
      value="amountAwarded|desc"
    >
      
            Highest amount first
        
    </option>
  </select>
</div>
`;
